lib/static/new-ui/components/UiModeHintNotification/index.module.css

```diff
@@ -0,0 +1,91 @@
+@keyframes notification-appear {
+    0% {
+        opacity: 0;
+        visibility: visible;
+        filter: blur(20px);
+    }
+
+    30% {
+        opacity: 1;
+    }
+
+    80% {
+        filter: blur(0px);
+        transform: scale(1.02);
+    }
+
+    100% {
+        filter: blur(0px);
+        transform: scale(1);
+        visibility: visible;
```

Comment:
Seems like "visibility: visible" does nothing here

Reply:
No, it's crucial here. When using animation-fill-mode forwards, setting visibility at the final stage forces element to stay visible after animation end. If we remove this line, notification will abruptly disappear as soon as animation ends.